src/models/src/messages/mod.rs

```diff
@@ -39,13 +39,24 @@ pub struct SlackMessageSender {
     pub display_as_bot: Option<bool>,
 }
 
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackRootMessage {
```

Comment:
`SlackRootMessage` - I find this is confusing name. In Slack API documentations there is a term as *parent message*:
https://api.slack.com/messaging/managing#threading
I'd suggest to use that instead of *root message*.
So, it would be something like `SlackParentMessageParams` (I added Params part to the name since we don't have real message content inside to make it more explicit)